build.sbt

```diff
@@ -817,45 +819,48 @@ lazy val `pubsub-scaladsl` = (project in file("pubsub/scaladsl"))
   )
   .configs(MultiJvm)
 
-lazy val `cluster-extensions` = (project in file("cluster/extensions"))
-  .dependsOn(`cluster-core`, logback % Test)
+lazy val `projection-core` = (project in file("projection/core"))
+  .dependsOn(
+    `cluster-core` % "compile;multi-jvm->multi-jvm",
```

Comment:
For some reason, `multi-jvm->multi-jvm` is not working the way I was expecting.

This module needs to use a class that is defined in `multi-jvm` folder in `cluster-core`. If I only compile `projection-core`, it will trigger the compilation of the dependency, but somehow it's not available when compiling the tests in `projection-core`.

@dwijnand, any idea about how this is not working? I checked the akka project and they do similar things. I guess there is something special in our configuration that makes it not to work.

Reply:
That was a stupid mistake from me in which I didn't see a missing import. 😞 

In any case, got some improvements suggestion from Dale on how to configure the MultiJvm tests and applied those.